### PR TITLE
Update Rule “dones-do-you-use-email-for-tasks-only-not-communication/rule”

### PR DESCRIPTION
**Pull request by AshAnil**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by chrisschultzssw — APPROVED** [review]
Good change!

**Review by Hona — DISMISSED** [review]
Remove the trailing slashes from netlify.

Happens with the ::: frontmatter blocks

File: rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md [hunk 1]
```diff
@@ -30,51 +28,53 @@ Email is a very powerful business tool. The main problem, however, is that for m
 
 Using your mailbox as a task list also saves you from having to use inefficient paper-based cards to track your tasks.
 
-::: email-template  
+::: email-template\
```
Comment (JackDevAU) [inline rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md:31]: Please remove the trailing slash's in this file  i.e.`\`  (This is a bug with Netlify 😟)

File: rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md [hunk 2]
```diff
@@ -30,51 +28,53 @@ Email is a very powerful business tool. The main problem, however, is that for m
...
 |          |     |
 | -------- | --- |
 | From:    | Tim |
 | To:      | Mark |
-| Subject: | What do you think? |  
+| Subject: | What do you think? |\
```
Comment (JackDevAU) [inline rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md:36]: here

File: rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md [hunk 3]
```diff
@@ -30,51 +28,53 @@ Email is a very powerful business tool. The main problem, however, is that for m
...
+### Hi Mark,
+
 I am not 100% happy , maybe we should look into making the menu better... 
 
 Tim
 
-:::  
-:::  
-::: bad  
+:::\
```
Comment (JackDevAU) [inline rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md:45]: here

File: rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md [hunk 4]
```diff
@@ -30,51 +28,53 @@ Email is a very powerful business tool. The main problem, however, is that for m
...
+:::\
+::: bad\
 Figure: Bad Example - A bad email is one that gives no clear action items and will probably lead to a back and forth thread (potentially with people misunderstanding each other) 
 :::
 
-::: email-template  
+::: email-template\
```
Comment (JackDevAU) [inline rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md:51]: here

File: rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md [hunk 5]
```diff
@@ -30,51 +28,53 @@ Email is a very powerful business tool. The main problem, however, is that for m
...
 1. Please move "Settings" into "Options" on the main menu
 
 Tim
 
-:::  
-:::  
-::: good  
+:::\
+:::\
+::: good\
```
Comment (JackDevAU) [inline rules/dones-do-you-use-email-for-tasks-only-not-communication/rule.md:69]: here

**Review by JackDevAU — CHANGES_REQUESTED** [review]

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**